assets/app/view/game/company.rb

```diff
@@ -65,14 +66,13 @@ def render
 
         props = {
           style: {
-            display: 'inline-block',
             cursor: 'pointer',
             border: 'solid 1px gainsboro',
             'border-radius': '10px',
             overflow: 'hidden',
             padding: '0.5rem',
             margin: '0.5rem 0.5rem 0 0',
-            width: '300px',
+            width: '19rem',
```

Comment:
is this going to work on all browsers?

Reply:
I can’t test ’em all (FF, Chrome + Safari/iOS worked fine), but 19rem + 1rem padding = 20rem = width of corps and player cards. So if nobody did complain about them until now I wouldn’t expect any problems.
If font-size (user-setting in browser!) is set to 16px all these cards are 320px wide (+1 px border each side)